examples/src/app/helpers/formatters.mjs

```diff
@@ -34,7 +34,11 @@ const getTypeScriptFunctionFromText = (text) => {
 };
 
 const getInnerFunctionText = (text) => {
-    text = text.substring(text.indexOf("{") + 1);
+    if (text.includes(': void {')) {
+        text = text.substring(text.indexOf(": void {") + 8);
```

Comment:
the `+8` bit here does not fill me with confidence. Perhaps this should be refactored to be more self-explanatory, or a comment should be added?

Reply:
Yeah it's definitely brittle, unfortunately I've found no great way to template these example files so I can extract various parts in a less brittle way. I'll try and update this to be more self explanatory though.